docs/topics/auth/default.txt

```diff
@@ -978,6 +978,19 @@ implementation details see :ref:`using-the-views`.
 
         The ``redirect_authenticated_user`` parameter was added.
 
+        .. warning::
```

Comment:
Since there's no other documentation about the parameter here, I wouldn't repeat the warning.